containerd_execution.go

```diff
@@ -137,7 +137,7 @@ func (t *createTask) executeCreateContainer(args ...string) (containerId string,
 	cmd.Stdout = stdout
 	cmd.Stderr = stdErr
 	if err = cmd.Run(); err != nil {
-		return "", err
+		return "", fmt.Errorf("%s: %w", stdErr.String(), err)
```

Comment:
I feel like printing the cmd err first and then the stdErr might make more sense? I'll leave it up to you though

Reply:
sure i can swap them